packages/mui-docs/src/NProgressBar/NProgressBar.js

```diff
@@ -38,52 +42,55 @@ function NProgressBar(props) {
   return (
     <NoSsr>
       {props.children}
-      <GlobalStyles
-        styles={(theme) => ({
-          '#nprogress': {
-            direction: 'ltr',
-            pointerEvents: 'none',
+      {/* Error: It seems you are interpolating a keyframe declaration (ikWivX) into an untagged string. This was supported in styled-components v3, but is not longer supported in v4 as keyframes are now injected on-demand. Please wrap your string in the css\`\` helper which ensures the styles are injected correctly. See https://www.styled-components.com/docs/api#css */}
```

Comment:
This is not related to the `styled-components` v6, I am checking in parallel the docs running with `styled-components` v5 in https://github.com/mui/material-ui/pull/39043 and the error is happening there as well.